README.md

```diff
@@ -1,56 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Math magicians
 
-> Description the project.
+> First project using react
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- React
+- VScode
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+coming soon
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+git clone link [https://github.com/BonkeGcobo/math-magician.git] 
+cd math-magician
```

Comment:
- [OPTIONAL] Please it will be nice to complete the steps to get the app running as this will be of help to non-technical users, steps like; `npm install`, `npm start`.